cmd/bridge/main.go

```diff
@@ -107,6 +107,14 @@ func main() {
 	fStatuspageID := fs.String("statuspage-id", "", "Unique ID assigned by statuspage.io page that provides status info.")
 	fDocumentationBaseURL := fs.String("documentation-base-url", "", "The base URL for documentation links.")
 
+	fAlermanagerURL := fs.String("alermanager-url", "", "Alertmanager monitoring URL.")
```

Comment:
You should make it clear in the description that it's the public URL, not the in-cluster URL (which we also use)

Reply:
No worries at all. Makes sense to change ad the `Public` word  👍